nextflow.config

```
@@ -71,7 +71,7 @@ params {
     multiqc_methods_description = null
 
     // Boilerplate options
-    outdir                     = null
+    outdir                     = './results'
```

```suggestion
    outdir                     = '.null'
```
default outdir should be `null`

---

Should this then also be changed to `null` as default in `nextflow_schema.json` ?
This was the linting error:

`nextflow_config: Config default value incorrect: params.outdir is set as ./results in nextflow_schema.json but is null in nextflow.config.`

---

Yes, it should also be `null` default in the schema